include/miopen/miopen.h

```diff
@@ -5793,6 +5794,95 @@ MIOPEN_EXPORT miopenStatus_t miopenSumForward(miopenHandle_t handle,
 // CLOSEOUT SUM DOXYGEN GROUP
 #endif
 
+#ifdef MIOPEN_BETA_API
+// Padreflection APIs
+/** @addtogroup pad
+ *
+ *  @{
+ */
+/*! @brief Add padding by the reflection of the tensor
+ *
+ * @param handle                   MIOpen handle (input)
+ * @param xDesc                    Tensor descriptor for data input tensor x (input)
+ * @param x                        Data tensor x (input)
+ * @param yDesc                    Tensor descriptor for output data tensor y (input)
+ * @param y                        Data tensor y (output)
+ * @param padding                  Padding array contain 1 element only (input)
+ * @param num_padding              Number of elements in padding, equals 1 or 4 (input)
+ * @return                         miopenStatus_t
+ */
+MIOPEN_EXPORT miopenStatus_t
+miopenPadReflection1dFwdContiguous(miopenHandle_t handle,
```

Comment:
It should be miopenPadReflectioFwd. Check if they are 1d or 2d, cont or non-cont and etc. in your solver and call the right kernel.

Reply:
Also applies to bwd. I'll resume the review after this fix.

Reply:
I have refactored the code, adding the appropriate checks in problemdescription and the solver in commit [5646e90](https://github.com/moreh-dev/MIOpen/pull/15/commits/5646e9052d08acb2e5da4a045e3ac931d3ac65fb)

Reply:
sorry I comment with a different github account, s4002304 is me Duong Le